src/components/elements/NoopReadingRenderer.tsx

```diff
@@ -0,0 +1,7 @@
+import * as Renderer from '../renderers/renderers';
+
```

Comment:
what's the purpose of this component?

Reply:
It's for when you don't want to render a particular type of sensor. It was a last minute change to try to not render the count total as a separate tile, but it didn't work as expected. We can iterate on this.  It may be better handled in a different way.